### ISSUE TITLE
Clone connection pool

### ISSUE THREAD

**Issue by tyhi** [prisma/quaint#52]
I really like quaint so far however I would like to use it in a web but I'm not able to currently as the connection pool looks like clone isn't implemented or it is private. Here's an example with actix-web:

```
let pool = Quaint::new("postgres:////user:password@host/database").await?;

actix_web::HttpServer::new(move || {
        actix_web::App::new()
            .data(pool.clone())
            .service(routes::routes())
    })
    .bind(format!("0.0.0.0:{}", port))?
    .run()
    .await?;

```

**Comment by pimeys** [comment]
Hmm, we could definitely allow cloning with an internal `Arc`, but you can also do it by yourself by just wrapping `Quaint` into an `Arc`.

**Comment by tomhoule** [comment]
This is implemented in #53 :tada: We'll make a patch release

**Comment by tomhoule** [comment]
The change was released as part of 0.1.5, closing this issue :)
